I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/story/week one/firstday.rpy", line 22, in script
    n "You sigh. Another day, another dollar. But today's gonna be different."
  File "game/story/week one/firstday.rpy", line 22, in script
    n "You sigh. Another day, another dollar. But today's gonna be different."
TypeError: missing a required argument: 'OVERWRITE_SAVE'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\main.py", line 652, in main
    run(restart)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\execution.py", line 958, in run_context
    context.run()
  File "game/story/week one/firstday.rpy", line 22, in script
    n "You sigh. Another day, another dollar. But today's gonna be different."
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "game/story/week one/firstday.rpy", line 22, in script
    n "You sigh. Another day, another dollar. But today's gonna be different."
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 2586, in execute
    Say.execute(self)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 623, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\exports\sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\character.py", line 1470, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\character.py", line 1117, in do_display
    display_say(who,
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\character.py", line 814, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 2218, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 2748, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\screen.py", line 480, in visit_all
    callback(self)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 2748, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\screen.py", line 491, in per_interact
    self.update()
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\screen.py", line 700, in update
    self.screen.function(**self.scope)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\parameter.py", line 455, in apply_arguments
    return parameters.apply(args or (), kwargs or {}, ignore_errors)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\parameter.py", line 307, in apply
    raise TypeError(msg) # from None
TypeError: missing a required argument: 'OVERWRITE_SAVE'

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.6.25022803
Aperture Dating Simulator 0.5.5alpha
Sun Jun 29 21:06:49 2025
